docs/CxIAST-Integration.md

```diff
@@ -0,0 +1,82 @@
+* [Suggested Flow](#suggestedFlow)
+* [Configuration](#configuration)
+* [Filters](#filters)
+* [Thresholds](#thresholds)
+* [Bug Trackers](#bugTrackers)
+
+## <a name="suggestedFlow">Suggested Flow</a>
+CxIAST can be integrated within a CI/CD pipeline using CxFlow.  
+A CxIAST scan starts automatically when an application with an attached CxIAST agent is being tested.  
+CxIAST scans can be stopped in two ways:
```

Comment:
CxIAST can be integrated within a CI/CD pipeline using CxFlow.  CxIAST scans start automatically when an application with an attached CxIAST agent is being tested.  To attach a CxIAST agent, please follow the following steps: [Insert steps here].
  
CxIAST scans can be stopped in two ways: